Makefile

```diff
@@ -85,6 +85,10 @@ install: all
 	sed "s!PREFIX!$(PREFIX)!g; s!VERSION!$(VERSION)!g" nsxiv.1 \
 		>$(DESTDIR)$(MANPREFIX)/man1/nsxiv.1
 	chmod 644 $(DESTDIR)$(MANPREFIX)/man1/nsxiv.1
+	@echo "INSTALL nsxiv.desktop"
+	install -Dt $(DESTDIR)$(PREFIX)/share/applications nsxiv.desktop
```

Comment:
We're not using `install` anymore, as it's not POSIX. Try `mkdir -p` and then `cp` instead.